lib/nanoc/cli.rb

```diff
@@ -110,6 +110,11 @@ def self.setup_commands
       cmd = load_command_at(cmd_filename)
       add_command(cmd)
     end
+
+    if defined?(Bundler)
+      # Discover external commands through Bundler
+      Bundler.require(:nanoc)
```

Comment:
What happens if no `nanoc` group exists? What if it is empty?

A change without tests puts me off, but at the same time, I have no idea how to test this in an automated fashion.